src/masternodes/rpc_customtx.cpp

```diff
@@ -185,6 +185,22 @@ class CCustomTxRpcVisitor : public boost::static_visitor<void>
         rpcInfo.pushKV("maxPrice", ValueFromAmount((obj.maxPrice.integer * COIN) + obj.maxPrice.fraction));
     }
 
+    void operator()(const CPoolSwapMessageV2& obj) const {
+        (*this)(obj.swapInfo);
```

Comment:
Unused statement.

Reply:
It used, it calls `void operator()(const CPoolSwapMessage& obj)`

Reply:
Makes sense now, I was not seeing the second round brackets as a call operator!